Makefile

```diff
@@ -26,6 +27,7 @@ push:
 test:
 	./gradlew cleanTest test
```

Comment:
Not running `./gradlew integTest` anywhere.   Consider running tests before the assemble